steps/helm/src/types/definition.ts

```diff
@@ -8,6 +8,10 @@ export const definitions = {
     is_service: {
       type: 'boolean'
     },
+    // TODO: Alias should only be avilable for is_service = true.
+    alias: {
```

Comment:
@FCQCarryOut kannst du das schema hier anpassen?

Reply:
done in b601318dc8b80215bc95eb8cc7fde3935c2918ad

Reply:
Noice, i removed todo.